src/modules/charge-versions/routes/charge-version-workflows.js

```diff
@@ -90,7 +89,8 @@ module.exports = {
         payload: {
           status: Joi.string(),
           chargeVersion: Joi.object(),
-          approverComments: Joi.string().allow(null)
+          approverComments: Joi.string().allow(null),
+          createdBy: Joi.object().optional()
```

Comment:
Would not require this if we use the internal calling user

Reply:
as mentioned above - unfortunately it is the only option with the current workflow/cron job solution